### DISCUSSION TITLE
Admin UI for db-scheduler library

### DISCUSSION THREAD

**Discussion by an0nh4x0r** [kagkarlsson/db-scheduler#385] [Ideas]
Do we have any support for the Admin UI for the db-scheduler library? 

Like internally using the scheduler-client to cancel, or update the execution time? 

If not then I'm happy to create a repository with maybe Angular and the latest version of bootstrap for the Admin portal.

**Comment by sweetcard** [comment]
That’s very cool!
Your excellent repository will be cool!😄

**Comment by kagkarlsson** [comment]
Hi! A UI would be useful, but it would have to be decoupled from the main-repo, i.e. live in a repo of its own. Additionally, I don't think I have the time to maintain a UI, at least not now. So feel free to create one using the `SchedulerClient` in an external repo of your choice :)

**Reply by an0nh4x0r** [reply]
Sure thanks a lot for the response. I'll create a repo today 🙂

Excited to work on this 🙂
